local-volume/provisioner/deployment/kubernetes/gce/provisioner_generated_gce_19.yaml

```diff
@@ -0,0 +1,64 @@
+---
+# Source: provisioner/templates/provisioner.yaml
+ 
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: local-provisioner-config 
+  namespace: default 
+data:
+  storageClassMap: |     
+    local-scsi:
```

Comment:
One more thing I'm considering: Instead of naming these storage classes "local-scsi" and "local-nvme", maybe "local-fast" and "local-faster" to be more portable across environments.

Reply:
I can see how this would be useful for portability, but not liking the "local-fast" and "local-faster" names. Hard to disambiguate, and if we go down this route what happens when we need a "local-fasterer"

Reply:
Maybe "standard" or "local-ssd".

Reply:
Yeah, the challenge with "local-ssd" is that on GCE, there are two kinds, scsi and nvme.  And who knows, maybe there will be more in the future.

Reply:
I'm going to keep the naming the way it is for now since I can't think of a better name.  Users can still generate their own specs with different storage class names if they want.

Reply:
Agreed. I also can't think of anything better.